vision/unit_tests/test_safe.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+
+class TestSafeSearchAnnotation(unittest.TestCase):
+    @staticmethod
+    def _get_target_class():
+        from google.cloud.vision.safe import SafeSearchAnnotation
+        return SafeSearchAnnotation
+
+    def test_safe_search_annotation(self):
+        from google.cloud.vision.likelihood import Likelihood
+        from unit_tests._fixtures import SAFE_SEARCH_DETECTION_RESPONSE
```

Comment:
Bumping that value doesn't bother me.

> I like having it all in the same place

It may just be a preference difference between us? ISTM that it can be said that individual tests are hard to reason about if the data is not in the test (independent of preference), but maybe I'm wrong?